ring.go

```diff
@@ -14,8 +13,9 @@ type ring struct {
 	endpoints []*HostInfo
 
 	// hosts are the set of all hosts in the cassandra ring that we know of
-	mu    sync.RWMutex
-	hosts map[string]*HostInfo
+	mu           sync.RWMutex
+	hosts        map[string]*HostInfo
```

Comment:
Please add doc comment that the key is hostID.

Reply:
Added

Reply:
Thanks!